log/log.go

```diff
@@ -39,10 +41,64 @@ func (f levelFlag) Set(level string) error {
 	return nil
 }
 
+func setSyslogFormatter(appname, facility string) error {
+	if appname != "" && facility != "" {
+		return nil
```

Comment:
Might be good to return an error here instead of nil. This code path is only reached if someone set `logger:syslog`. We should tell them that appname and facility are mandatory.

If you think it makes sense, we could alternatively set facility to some default value.